src/pseudopeople/constants/metadata.py

```diff
@@ -79,6 +79,8 @@ class __DateFormats:
     "WISCONSIN": "WI",
     "WYOMING": "WY",
     "DISTRICT OF COLUMBIA": "DC",
+    # US for sample data
+    "UNITED STATES": "US",
```

Comment:
Doesn't this mean they'll be able to use "US" to filter the full dataset?

Reply:
Yes and then it would return an empty dataframe and raise a ValueError.